cross/usb-serial/Makefile

```diff
@@ -19,7 +19,7 @@ include ../../mk/spksrc.cross-cc.mk
 myCompile:
 	@mkdir -p $(WORK_DIR)/$(PKG_DIR)
 	@echo "obj-m += usbserial.mod.o" > $(WORK_DIR)/$(PKG_DIR)/Makefile
-	@echo "obj-m += usb-serial.o" > $(WORK_DIR)/$(PKG_DIR)/Makefile
+	@echo "obj-m += usb-serial.o" >> $(WORK_DIR)/$(PKG_DIR)/Makefile
```

Comment:
I did not verify whether it matters that usebserial.mod.o was missing in the generated Makefile